packages/client/src/views/Settings/ChangePhonePage.tsx

```diff
@@ -0,0 +1,479 @@
+  goBack as goBackAction
+} from '@client/navigation'
+import { BackArrow } from '@opencrvs/components/lib/icons'
+import {
+  SysAdminContentWrapper,
+  SysAdminPageVariant
+} from '@client/views/SysAdmin/SysAdminContentWrapper'
+import { EMPTY_STRING } from '@client/utils/constants'
+import {
+  isAValidPhoneNumberFormat,
+  phoneNumberFormat
+} from '@client/utils/validate'
+import { Mutation } from 'react-apollo'
+import gql from 'graphql-tag'
+import { get } from 'lodash'
+import { getCurrentUserScope } from '@client/utils/authUtils'
+
+const Container = styled.div`
+  ${({ theme }) => theme.shadows.mistyShadow};
+  color: ${({ theme }) => theme.colors.copy};
+  background: ${({ theme }) => theme.colors.white};
+  padding: 40px 77px;
+  margin: 36px auto;
+  width: 1140px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    margin: 0;
+    padding: 24px 0;
+    width: 100%;
+    min-height: 100vh;
+    margin-top: 0;
+    box-shadow: 0 0 0 rgba(0, 0, 0, 0);
+  }
+`
+const StyledPrimaryButton = styled(PrimaryButton)`
+  display: absolute;
+  width: 115px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    width: 100%;
+    margin-top: 24px;
+  }
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    margin-top: 24px;
+  }
+`
+const HalfWidthInput = styled(TextInput)`
+  width: 271px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    width: 100%;
+  }
+`
+const FormSectionTitle = styled.h4`
+  ${({ theme }) => theme.fonts.h4Style};
+  color: ${({ theme }) => theme.colors.copy};
+  margin-top: 0px;
+  margin-bottom: 16px;
+`
+
+const SettingsTitle = styled.div`
+  ${({ theme }) => theme.fonts.h1Style};
+  height: 72px;
+  margin-left: 16px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    display: none;
+  }
+`
+
+const Content = styled.div`
+  display: flex;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    flex-direction: column-reverse;
+  }
+`
+const Left = styled.div`
+  margin: 0 16px;
+  flex-grow: 1;
+`
+const Right = styled.div`
+  display: flex;
+  padding-top: 80px;
+  margin-left: 112px;
+  & .desktop {
+    display: block;
+  }
+  & .tablet {
+    display: none;
+  }
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    padding-top: 0;
+    margin-left: 24px;
+    & .desktop {
+      display: none;
+    }
+    & .tablet {
+      display: block;
+    }
+  }
+`
+const Row = styled.div`
+  display: flex;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    margin-bottom: 20px;
+  }
+`
+const Field = styled.div`
+  margin-bottom: 30px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    margin-bottom: 0px;
+  }
+`
+const Message = styled.div`
+  margin-bottom: 16px;
+`
+const Label = styled.label`
+  margin-bottom: 8px;
+`
+const InvalidPhoneNumber = styled.div`
```

Comment:
![image](https://user-images.githubusercontent.com/1206987/150801352-eafe9f85-4c93-46f1-bb17-c149884c5d4a.png)

The font seems to be broken here

Reply:
![image](https://user-images.githubusercontent.com/1206987/150801562-0857dea9-8615-4915-9da9-0d3a2f03ea8f.png)

Found a couple of other places as well. Try searching the code base with 

```
${({ theme }) => theme.fonts.semiBoldFont};
```

Reply:
This bug is fixed now. Thanks for the review.